src/libraries/System.Numerics.Tensors/tests/TensorPrimitivesTests.cs

```diff
@@ -20,7 +20,7 @@ public static partial class TensorPrimitivesTests
             TensorLengths.Concat(new object[][] { [0] });
 
         public static IEnumerable<object[]> TensorLengths =>
-            from length in Enumerable.Range(1, 128)
+            from length in Enumerable.Range(1, 256)
```

Comment:
This is good, but it's also going to double the number of test cases we're running.  Not a big deal except that on netfx it seems to struggle with the theory count.  We should probably subsequently change some of those theories to be loops, e.g. instead of:
```C#
[Theory]
[MemberData(nameof(TensorLengths))]
public void Foo(int tensorLength)
{
    ...
}
```
do:
```C#
[Fact]
public void Foo()
{
    foreach (int tensorLength in TensorLengths)
    {
        ...
    }
}
```

Reply:
I'll handle this in the immediately following PR covering binary/ternary.